usr/share/rear/finalize/GNU/Linux/250_migrate_disk_devices_layout.sh

```diff
@@ -32,16 +32,7 @@ local restored_file=""
 # the funny [] around the first letter make sure that shopt -s nullglob removes this file from the list if it does not exist
 # the files without a [] are mandatory, like fstab FIXME: but below there is [e]tc/fstab not etc/fstab - why?
 
-for restored_file in [b]oot/{grub.conf,menu.lst,device.map} [e]tc/grub.* [b]oot/grub/{grub.conf,menu.lst,device.map} \
-                     [b]oot/grub2/{grub.conf,grub.cfg,menu.lst,device.map} \
-                     [e]tc/sysconfig/grub [e]tc/sysconfig/bootloader \
-                     [e]tc/lilo.conf \
-                     [e]tc/yaboot.conf \
-                     [e]tc/mtab [e]tc/fstab \
-                     [e]tc/mtools.conf \
-                     [e]tc/smartd.conf [e]tc/sysconfig/smartmontools \
-                     [e]tc/sysconfig/rawdevices \
-                     [e]tc/security/pam_mount.conf.xml [b]oot/efi/*/*/grub.cfg
```

Comment:
A comment
```
# The variable expansion is deliberately not quoted in order to perform
# pathname expansion on the variable value.
```
should be added here tho have it same here
as in finalize/GNU/Linux/280_migrate_uuid_tags.sh

Reply:
Addressed in 8265c9c43e958309c1030270bb111e579e3089f8